.circleci/config.yml

```diff
@@ -548,11 +573,23 @@ workflows:
               only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
             branches:
               only: unsigned
+      - build-ios-iphonesimulator:
+          context: mattermost-mobile-unsigned
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
+            only:
+                - unsigned
+                - /^iphonesimulator-.*/
```

Comment:
Added so that we can build a version of iphonesimulator during RC that can run for RainforestQA.